remix.init/remix.config.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Is this needed as this is a js file?

Reply:
I think I had it because when it’s a TS project. I could try an `/* eslint-env */` at the top of the file instead.

Reply:
Hmm, no issues even with a TS project. I'll go ahead and remove it.